{
  "artifacts": [],
  "command": "my_main",
  "experiment": {
    "base_dir": "/home/zpp/graduation_design/pymarl_base/pymarl-master/src",
    "dependencies": [
      "munch==2.5.0",
      "numpy==1.19.5",
      "PyYAML==5.4.1",
      "sacred==0.8.2",
      "torch==1.7.1"
    ],
    "mainfile": "main.py",
    "name": "pymarl",
    "repositories": [],
    "sources": [
      [
        "main.py",
        "_sources/main_758fbd5734579733d1a65a9522fb1179.py"
      ],
      [
        "run.py",
        "_sources/run_0faa17e7222dddde755a7c40d4e96600.py"
      ],
      [
        "utils/__init__.py",
        "_sources/__init___d41d8cd98f00b204e9800998ecf8427e.py"
      ],
      [
        "utils/logging.py",
        "_sources/logging_2c814fe775935963fe055f6f37191d6e.py"
      ]
    ]
  },
  "fail_trace": [
    "Traceback (most recent call last):\n",
    "  File \"/home/zpp/anaconda3/envs/Smac/lib/python3.8/site-packages/sacred/config/captured_function.py\", line 42, in captured_function\n    result = wrapped(*args, **kwargs)\n",
    "  File \"src/main.py\", line 35, in my_main\n    run(_run, config, _log)\n",
    "  File \"/home/zpp/graduation_design/pymarl_base/pymarl-master/src/run.py\", line 48, in run\n    run_sequential(args=args, logger=logger)\n",
    "  File \"/home/zpp/graduation_design/pymarl_base/pymarl-master/src/run.py\", line 175, in run_sequential\n    evaluate_sequential(args, runner, buffer, learner)\n",
    "  File \"/home/zpp/graduation_design/pymarl_base/pymarl-master/src/run.py\", line 83, in evaluate_sequential\n    weights_info = learner.train(episode_sample, runner.t_env, episode_num = 0, get_info = True)\n",
    "  File \"/home/zpp/graduation_design/pymarl_base/pymarl-master/src/learners/q_learner.py\", line 101, in train\n    weights_info = self.mixer.get_weights_info(chosen_action_qvals, batch[\"state\"][:, :-1], feats_ls, batch[\"adjacency_matrix\"][:, :-1])\n",
    "UnboundLocalError: local variable 'feats_ls' referenced before assignment\n"
  ],
  "heartbeat": "2021-12-02T05:25:41.400078",
  "host": {
    "ENV": {},
    "cpu": "Intel(R) Xeon(R) Platinum 8280 CPU @ 2.70GHz",
    "gpus": {
      "driver_version": "460.39",
      "gpus": [
        {
          "model": "GeForce RTX 3090",
          "persistence_mode": false,
          "total_memory": 24268
        },
        {
          "model": "GeForce RTX 3090",
          "persistence_mode": false,
          "total_memory": 24268
        },
        {
          "model": "GeForce RTX 3090",
          "persistence_mode": false,
          "total_memory": 24268
        },
        {
          "model": "GeForce RTX 3090",
          "persistence_mode": false,
          "total_memory": 24268
        }
      ]
    },
    "hostname": "admin001-SYS-7049GP-TRT",
    "os": [
      "Linux",
      "Linux-5.8.0-49-generic-x86_64-with-glibc2.10"
    ],
    "python_version": "3.8.5"
  },
  "meta": {
    "command": "my_main",
    "options": {
      "--beat-interval": null,
      "--capture": null,
      "--comment": null,
      "--debug": false,
      "--enforce_clean": false,
      "--file_storage": null,
      "--force": false,
      "--help": false,
      "--loglevel": null,
      "--mongo_db": null,
      "--name": null,
      "--pdb": false,
      "--print-config": false,
      "--priority": null,
      "--queue": false,
      "--s3": null,
      "--sql": null,
      "--tiny_db": null,
      "--unobserved": false,
      "COMMAND": null,
      "UPDATE": [
        "env_args.map_name=MMM2"
      ],
      "help": false,
      "with": true
    }
  },
  "resources": [],
  "result": null,
  "start_time": "2021-12-02T05:23:20.607185",
  "status": "FAILED",
  "stop_time": "2021-12-02T05:25:41.402022"
}